stdlib/LinearAlgebra/test/cholesky.jl

```diff
@@ -399,4 +405,26 @@ end
     @test B ≈ Matrix(C)
 end
 
+@testset "CholeskyPivoted and Factorization" begin
+    A = randn(8,8)
+    B = A'A
+    C = cholesky(B, Val(true), check=false)
+    @test CholeskyPivoted{eltype(C)}(C) === C
+    @test Factorization{eltype(C)}(C) === C
+    @test Array(CholeskyPivoted{complex(eltype(C))}(C)) ≈ Array(cholesky(complex(B), Val(true), check=false))
+    @test Factorization{complex(eltype(C))}(C) ≈ cholesky(complex(B), Val(true), check=false)
```

Comment:
The latest test failures are real and related to this line. Given the three tests above, you may simply remove this one without reducing code coverage, I'd say.